plugins/check_smtp.c

```diff
@@ -288,16 +288,17 @@ main (int argc, char **argv)
 			printf("%s", buffer);
 		}
 
-#  ifdef USE_OPENSSL
-		  if ( check_cert ) {
+		}
+#endif
```

Comment:
Please add `/* HAVE_SSL */` here, so it is a little bit easier to see what ends here

Reply:
Sure! I added a separate commit to preserve the original backported commits, though.